src/main/java/core/basesyntax/AreaCalc.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalc {
```

Comment:
```suggestion
public interface AreaCalculator {
```
avoid shortenings please